Dangerfile

```diff
@@ -1,2 +1,52 @@
-# Lint files with Swiftlint. Check .swiftlint.yml for config
-swiftlint.lint_files
\ No newline at end of file
+
+
+# Add a link to the Bugzilla bug
+
+# for debugging uncomment out these 2 lines
+# require 'pry'
+# binding.pry
+
+# Run swiftlint
+swiftlint.lint_files
+
+# Localized Strings check
+changedFiles = (git.added_files + git.modified_files).select{|file| file.end_with?(".swift")}
+changedFiles.select{|file| file != "Client/Frontend/Strings.swift" }.each do |changed_file|
+  addedLines = git.diff_for_file(changed_file).patch.lines.select{ |line| line.start_with?("+") }
+  if addedLines.select{ |line| line.include?("NSLocalizedString") }.count != 0
+    warn("NSLocalizedString should only be added to Strings.swift")
+    break # We only need to show the warning once
+  end
+end
+
+# Add a friendly reminder for Sentry
```

Comment:
The consequence (and likelihood) of misunderstanding what a Sentry log message does is too low to warrant a check outside of a regular code review.

Reply:
+1 to the the other checks you added which I think could catch some legit probs